### ISSUE TITLE
Add an easy way to update notion

### ISSUE THREAD

**Issue by sphinxc0re** [notion-cli/notion#269]

**Comment by rwjblue** [comment]
Totally agree that we need a better solution here!

To date I've been doing:

```
rm -rf ~/notion/bin ~/notion/cache ~/notion/catalog.toml ~/notion/config.toml ~/notion/launchbin ~/notion/launchscript ~/notion/load.sh ~/notion/notion ~/notion/tmp ~/notion/tools
```

And re-running the installer...

**Comment by rwjblue** [comment]
Just to note, you can actually do the following now:

```
rm -rf $NOTION_HOME
curl -sSLf https://get.notionjs.com | bash
```

**Comment by Stanzilla** [comment]
Think of the Windows users as well, please :)

**Comment by charlespierce** [comment]
@Stanzilla definitely! The story for upgrades is actually a little more straightforward on Windows, since the installer already has support built-in.

We have a windows installer, but at the moment it’s not signed. We’re looking into getting a code-signing certificate so we can make it a little nicer.

**Comment by ProLoser** [comment]
Actually we need a way to easily transition to `volta` now. I am unable to find instructions on what should be uninstalled or installed.

**Comment by zhaoyao91** [comment]
hi, what is the progress of this issue?

**Comment by charlespierce** [comment]
Hi @zhaoyao91, there hasn't been any change: You can update by re-running the installer (either the pre-packaged one for Windows or the install script `curl https://get.volta.sh | bash` for Linux / MacOS.

**Comment by chawyehsu** [comment]
If anyone is using volta on Windows, currently there is an easy way to update volta. You can install volta via [Scoop](https://github.com/volta-cli/volta/issues/743#issuecomment-664867071), and just type `scoop update volta` to upgrade. I've been comfortable with it for a few months (since volta 0.8.x).
